### DISCUSSION TITLE
Bulk Collection Import Support

### DISCUSSION THREAD

**Discussion by RyanObray** [usebruno/bruno#1541] [Ideas]
I'm currently migrating from Postman and I have many collections. Something I'm finding quite painful is not being able to select multiple Postman exports at a time, forcing me to import one collection at a time. It would be very helpful for the import file selector to allow multi-select and for the import to process all collections selected at once.

**Comment by mikeedjones** [comment]
For openAPI specs I've put together https://github.com/usebruno/bruno/pull/1685 - think the same approach would work for postman

**Comment by helloanoop** [comment]
Latest version of Bruno supports importing multiple postman collections at a time.
